### PR TITLE
Add `Error handler` example

### PR DESCRIPTION
**Pull request by ppcano**
## What?

Add `Error handler` example

## Checklist

Please fill in this template:
- [x] I have used a meaningful title for the PR.
- [x] I have described the changes I've made in the "What?" section above.
- [x] I have performed a self-review of my changes.
- [x] I have run the `make docs` command locally and verified that the changes look good.

## Related PR(s)/Issue(s)

[Logging in k6](https://github.com/grafana/k6-docs/issues/327)

### CODE REVIEW

File: docs/sources/next/examples/error-handler.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: 'Error handler'
+excerpt: |
+  Using a custom error handler to store errors in custom metrics or logs.
```
Comment (heitortsergent) [inline docs/sources/next/examples/error-handler.md:4]:
```suggestion
description: Using a custom error handler to store errors in custom metrics or logs.
```

File: docs/sources/next/examples/error-handler.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+# Error handler
+
+When encountering errors from the backend, it's often necessary to gather additional error information, such as error messages, tracing data, or request and response bodies. Our first suggestion is to leverage your observability solution to find these errors. However, capturing error details directly in k6 can also be useful for troubleshooting.
+
+In k6, there are two common approaches to store additional error information:
+
+- Console logs and output [k6 logs to Loki](https://k6.io/blog/using-loki-to-store-and-query-k6-logs/), a file, or use [Grafana Cloud k6](https://grafana.com/docs/grafana-cloud/k6/analyze-results/inspect-test-results/inspect-logs/).
+- Using [Tags](https://grafana.com/docs/k6/<K6_VERSION>/using-k6/tags-and-groups/) in a [custom counter metric](https://grafana.com/docs/k6/<K6_VERSION>/using-k6/metrics/create-custom-metrics) to track error data.
+
+When deciding what error data to store, consider whether it is an general or specific error, and be aware that high-load tests may generate a substantial volume of error data.
```
Comment (heitortsergent) [inline docs/sources/next/examples/error-handler.md:17]:
```suggestion
When deciding what error data to store, consider whether it's a general or specific error, and be aware that high-load tests may generate a substantial volume of error data.
```

File: docs/sources/next/examples/error-handler.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+Below is an example using an ErrorHandler to log error information. It accepts a callback that instructs how to log errors. The example provides instructions for both previously mentioned options: console logs and custom metrics.
```
Comment (heitortsergent) [inline docs/sources/next/examples/error-handler.md:19]:
```suggestion
Below is an example using an `ErrorHandler` class to log error information. It accepts a callback that instructs how to log errors. The example provides instructions for both previously mentioned options: console logs and custom metrics.
```

File: docs/sources/next/examples/error-handler.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
+---
+title: 'Error handler'
+excerpt: |
+  Using a custom error handler to store errors in custom metrics or logs.
+weight: 25
+---
+
+# Error handler
+
+When encountering errors from the backend, it's often necessary to gather additional error information, such as error messages, tracing data, or request and response bodies. Our first suggestion is to leverage your observability solution to find these errors. However, capturing error details directly in k6 can also be useful for troubleshooting.
```
Comment (heitortsergent) [inline docs/sources/next/examples/error-handler.md:10]:
```suggestion
When encountering errors from an application's backend, gathering additional error information, such as error messages, tracing data, or request and response bodies, is often necessary. The initial suggestion is to leverage your observability solution to find these errors. However, capturing error details directly in k6 can also be useful for troubleshooting.
```

**Review by heitortsergent — APPROVED** [review]
Left some very minor edits, LGTM overall! 🚀

**Comment by heitortsergent** [comment]
Ah I forgot to add this to review @ppcano, can you add this update to v0.49 as well? Or if you want me to do it, just let me know. 🙇

**Comment by ppcano** [comment]
@heitortsergent, I ported the changes to `v0.49` and applied your suggestions. This should now be ready.

**Pull request merged**